images/flaky-test-retryer/retryer.yaml

```diff
@@ -0,0 +1,49 @@
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: flaky-test-retry-bot
+spec:
+  template:
+    metadata:
+      labels:
+        app: flaky-test-retry-bot
+    spec:
+      containers:
+        image: gcr.io/trevorfarrelly-knative-2019/flaky-test-retryer:latest
+        imagePullPolicy
+        command: "/flaky-test-retryer"
+        args:
+        - "--service-account=/etc/test-account/service-account.json"
+        #- "--github-account=/etc/flaky-test-reporter-github-token/token"
```

Comment:
I think it makes sense to use flaky-test-reporter robot, since it's currently very lightly used, and this tool deals with PRs only and flaky-test-reporter deals with issues only, they won't intersect with each other

Reply:
done.